src/Hive/Hive.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFrameworks>net5.0</TargetFrameworks>
```

Comment:
Is this necessary/what changes between these two?

Reply:
*Strictly* the change isn't necessary, but it means that during the build invocation `TargetFramework` is set globally, no matter where in evaluation you are, which is rather helpful. It means, among other things, that I can access `TargetFramework` in projects imported before that `PropertyGroup`.